src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs

```diff
@@ -131,6 +131,7 @@ private void EndFunction(FunctionInstanceLogEntry item)
             }
 
             _metrics.EndEvent(invokeLatencyEvent);
+            _hostMetrics.TrackFunctionExecutionDuration(item.Duration.TotalSeconds, functionName);
```

Comment:
Do we want to record seconds or milli seconds?

Reply:
seconds